pkg/reconciliation/reconcile_racks.go

```diff
@@ -1783,33 +1782,34 @@ func (rc *ReconciliationContext) startCassandra(endpointData httphelper.CassMeta
 		}
 	}
 
-	var err error
-
-	if shouldReplacePod && replaceAddress != "" {
-		// If we have a replace address that means the cassandra node did
-		// join the ring previously and is marked for replacement, so we
-		// start it accordingly
-		rc.Recorder.Eventf(rc.Datacenter, corev1.EventTypeNormal, events.StartingCassandraAndReplacingNode,
-			"Starting Cassandra for pod %s to replace Cassandra node with address %s", pod.Name, replaceAddress)
-		err = mgmtClient.CallLifecycleStartEndpointWithReplaceIp(pod, replaceAddress)
-	} else {
-		// Either we are not replacing this pod or the relevant cassandra node
-		// never joined the ring in the first place and can be started normally
-		rc.Recorder.Eventf(rc.Datacenter, corev1.EventTypeNormal, events.StartingCassandra,
-			"Starting Cassandra for pod %s", pod.Name)
-		err = mgmtClient.CallLifecycleStartEndpoint(pod)
-	}
+	go func(pod *corev1.Pod) {
```

Comment:
I know, late comment, but it would have been great to see a comment here why this needs to run concurrently

Reply:
It's a background task to allow cass-operator to process next command. The Reconcile pattern should never block itself.